en/components/lineargauge.md

```diff
@@ -9,7 +9,9 @@ In Ignite UI for Angular, use the Linear Gauge component to see a simple display
 
 ### Demo
 
-This gauge supports only one scale, one set of tick marks and one set of labels. However, all the changes made to the gauge have animation enhancements. This animation is easily customizable by setting the `transitionDuration` property. You can achieve numerous needle shapes by fine tuning its shape parameters. The following sample demonstrates how setting multiple properties on the same gauge can transform it to completely different gauge.
+The linear gauge component allows for visualizing data in the form of a linear gauge. It provides a simple and concise view of a value compared against a scale and one or more ranges. It supports one scale, one set of tick marks and one set of labels. The component has also a built-in support for animated transitions. This animation is easily customizable by setting the `transitionDuration` property. The features of the linear gauge component include configurable orientation and direction, configurable visual elements such as the needle, and more.
+
```

Comment:
perhaps, we should have a new section/sample that shows orientation and direction properties?